lib/commands/env.js

```diff
@@ -351,4 +351,80 @@ env._printEnv = function (env) {
   xervo.io.print(env.name.yellow + ' = ' + env.value.grey)
 }
 
+// Find the index for the specified env var in the specified env var array.
+//    Returns -1 if not found.
+//
+var envVarInArray = function (envVars, name) {
+  for (var idx = 0; idx < envVars.length; idx++) {
+    if (envVars[idx].name.toUpperCase() === name.toUpperCase()) return idx
+  }
+
+  return -1
+}
+
+// ------------------------------------------------------------------------------
+/**
+ * Displays a prompt for choosing an envVar from a list.
+ * @param {Array} envVars Array of a project envVars.
+ * @param {String} [envVar] Default selection.
+ * @param {Function} cb
+ */
+// ------------------------------------------------------------------------------
+env.chooseEnvVarPrompt = function (envVars, envVar, cb) {
+  var envVarIndex = -1
+
+  if (typeof envVar === 'function') {
+    cb = envVar
+    envVar = null
+  }
+  // A default selection has been made. See if a match exists in the collection.
+  if (typeof envVar === 'string' && envVar !== '') {
+    envVarIndex = envVarInArray(envVars, envVar)
+    if (envVarIndex >= 0) return cb(null, envVars[envVarIndex])
+
+    // No match was found. Error out.
+    return cb({ code: 'NO_MATCHING_ENV_VAR' })
+  }
+
+  if (envVars.length === 1) {
+    var selectedEnvVar = envVars[0]
+    xervo.io.prompt.get([{
+      name: 'confirm',
+      description: 'Are you sure you want to use environment veriable ' + selectedEnvVar.name.data + '?',
+      message: 'Acceptable response is "yes" or "no".',
+      pattern: /^[yntf]{1}/i,
+      default: 'yes'
+    }], function (err, result) {
+      if (err) {
+        return error.handlePromptError(err, cb)
+      }
+      var y = /^[yt]{1}/i.test(result.confirm)
```

Comment:
This should be handled by the [`pattern`](https://github.com/XervoIO/cli/pull/164/files#diff-e14694dea7b084b01a98db754925e12eR395) specified right?